src/components/Pocket/PhonePocket.js

```diff
@@ -25,20 +29,22 @@ const Pocket = ({ id, pocketState, onPressAction }) => (
               : strings.pocketButtonEditWeightText
           }
           textStyle={styles.tabletButtonText}
+          onPress={openWeightEditModal}
         />
       </View>
+      <View style={styles.containerEditTablet}>
```

Comment:
This is the PhonePocket component but the style is named "Tablet". It still renders ok.

Reply:
At the end it seemed that whole View was unnecessary